sycl/test-e2e/OneapiDeviceSelector/cpu_gpu_acc.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: cpu, gpu, accelerator, opencl
+// REQUIRES: any-device-is-cpu, any-device-is-gpu, any-device-is-accelerator, any-device-is-opencl
```

Comment:
Same here, this should not be an E2E test. This comment can be applied to some other tests as well, but I will repeat it on those which are the first candidates for the rewrite.